HW_1/task_3.py

```diff
@@ -0,0 +1,32 @@
+# 3. Напишите код, который запрашивает число и сообщает
+# является ли оно простым или составным. Используйте правило для проверки:
+# “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+
+
+res = ''
+min_num = 0
+max_num = 100000
+
+while True:
+    n = int(input('Введите число от 0 до 100000: '))
+    if n < min_num or n > max_num:
+        print('Введено число меньше 0 или больше 100000.')
+        continue
+    else:
+        break
```

Comment:
Но в этом случае `continue` нужен
```suggestion
    if n < min_num or n > max_num:
        print('Введено число меньше 0 или больше 100000.')
        continue
    
    break
```

Reply:
Как вариант
```suggestion
    if min_num <= n <= max_num:
        break

    print('Введено число меньше 0 или больше 100000.')
```